### PR TITLE
bots: Update documentation and add HACKING file

### PR DESCRIPTION
**Pull request by stefwalter**
This gets us a bit better documentation for how the bots
work.

### CODE REVIEW

File: bots/HACKING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Hacking on the Cockpit Bots
+
+These are automated bots and testing that works on the Cockpit project. This
+includes updating operating system images, bringing in changes from other
+projects, releasing Cockpit and more.
+
+## Environment for the bots
+
+The bots work in containers that are built in the [cockpituous](https://github.com/cockpit-project/cockpituous)
+repository. New dependencies should be added there in the ```tests/Dockerfile```
```
Comment (fabiand) [inline bots/HACKING.md]: I'm not seeing this Dockerfile.

Comment (mareklibra) [reply bots/HACKING.md]: It's in the cockpituous project, https://github.com/cockpit-project/cockpituous/blob/master/tests/Dockerfile

Comment (stefwalter) [reply bots/HACKING.md]: Yup, that's right.

Comment (stefwalter) [reply bots/HACKING.md]: Updated to be more clear.

File: bots/HACKING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+file in that repository.
+
+## Invoking the bots
+
+ # The containers in the `cockpitous` repository, invoke the ```.tasks``` file
```
Comment (martinpitt) [inline bots/HACKING.md]: Drop comma after "repository". This line and the following two prefixed with `#`, which looks really strange in Markdown (headings) and in plaintext too. Triple ` are unusual, please use single `

Comment (stefwalter) [reply bots/HACKING.md]: Fixed

**Comment by stefwalter** [comment]
Now that this is approved by @martinpitt I'll merge it. We're "scaled down" on bot activity, and I want to avoid testing this everywhere. It's just an update of documentation markdown files.

**Comment by martinpitt** [comment]
@stefwalter: Right, I'm fine with merging; I just thought we might use this PR to debug the bots, but you already did.

**Pull request merged**